state-sync/state-sync-v2/data-streaming-service/src/data_notification.rs

```diff
@@ -38,6 +38,8 @@ pub enum DataPayload {
 pub enum DataClientRequest {
     AccountsWithProof(AccountsWithProofRequest),
     EpochEndingLedgerInfos(EpochEndingLedgerInfosRequest),
+    NewTransactionOutputsWithProof(NewTransactionOutputsWithProofRequest),
```

Comment:
I'd prefer to keep it as it is simply because the name is consistent with what we added in the previous PR (i.e., on the storage side). Happy to update it in the future if we feel strongly though.